nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateRecord.java

```diff
@@ -180,6 +180,24 @@
         .defaultValue("true")
         .required(true)
         .build();
+    static final PropertyDescriptor MAX_VALIDATION_DETAILS_LENGTH = new PropertyDescriptor.Builder()
+        .name("maximum-validation-details-length")
+        .displayName("Maximum Validation Details Length")
+        .description("Specifies the maximum number of characters that validation details value can have. Any characters beyond the max will be truncated.")
```

Comment:
Just for completeness, it would be good to mention in this description that this property is only used if `Validation Details Attribute Name` is set. Also since it's being evaluated at the same time as the attribute name, what do you think about supporting FlowFile attributes for the expression evaluation?

Reply:
You're right!

Reply:
You're right!